Source/DafnyCompilers/Interp.dfy

```diff
@@ -468,41 +468,41 @@ module Interp {
     requires !op.MemberSelect?
   {
     match op
-      case BVNot => :- Need(v0.BitVector?, Invalid(expr));
-        Success(V.BitVector(v0.width, Math.IntPow(2, v0.width) - 1 - v0.value))
-      case BoolNot => :- Need(v0.Bool?, Invalid(expr));
-        Success(V.Bool(!v0.b))
-      case SeqLength => :- Need(v0.Seq?, Invalid(expr));
-        Success(V.Int(|v0.sq|))
-      case SetCard => :- Need(v0.Set?, Invalid(expr));
-        Success(V.Int(|v0.st|))
-      case MultisetCard => :- Need(v0.Multiset?, Invalid(expr));
-        Success(V.Int(|v0.ms|))
-      case MapCard => :- Need(v0.Map?, Invalid(expr));
-        Success(V.Int(|v0.m|))
+    case BVNot => :- Need(v0.BitVector?, Invalid(expr));
+                     Success(V.BitVector(v0.width, Math.IntPow(2, v0.width) - 1 - v0.value))
+    case BoolNot => :- Need(v0.Bool?, Invalid(expr));
+                       Success(V.Bool(!v0.b))
+    case SeqLength => :- Need(v0.Seq?, Invalid(expr));
```

Comment:
could we have a linebreak after `=>`? Why is `Success` aligned with the RHS of `:-`?

Reply:
The scope of the formatter does not include adding newlines.
Success is now aligned with :- and nots its RHS